lom/src/lib/clib/lom_clib.go

```diff
@@ -0,0 +1,254 @@
+package main
+
+import (
+    "C"
+    "encoding/json"
+    "fmt"
+    . "lom/src/lib/lomcommon"
+    . "lom/src/lib/lomipc"
+    "lom/src/engine"
+)
+
+
+/*
+ * C-bindings for non-go clients.
+ *
+ * A simple wrapper to corresponding APIs
+ */
+
+func TestCall() {
```

Comment:
What is this func for ? Is it possible to mock the functions using testify or gomock ?

Reply:
Guess, some left behind temp one. Removed.